source/utils/types.ts

```diff
@@ -67,6 +69,10 @@ export interface Response extends http.IncomingMessage {
 	request: { options: NormalizedOptions };
 }
 
+export interface ResponseObject extends ResponseLike {
```

Comment:
Is the naming okay?

Reply:
Yes, but I think it would be better to fix `ResponseLike` to have all the expected response properties.

Reply:
Sure, but how to deal with RFC 7234?

Reply:
What about it?

Reply:
We can't cache the `remoteAddress` with `ResponseLike` see https://github.com/lukechilds/cacheable-request/issues/86

Reply:
I still don't see how that's related to the types though. https://github.com/lukechilds/cacheable-request/issues/86 is about caching.

Reply:
But I think that lukechilds wont extend `ResponseLike` because of the RFC.

Reply:
That has nothing to do with the RFC. `ResponseLike` is meant to be exactly like a real response: https://github.com/lukechilds/responselike (It's a different module too).